src/routes/_components/dialog/components/ImageDialog.html

```diff
@@ -10,8 +10,7 @@
       class="image-modal-dialog-autoplay-video"
       aria-label="Animated GIF: {description || ''}"
       style="background-image: url({poster}); "
-      {width}
-      {height}
+      {style}
```

Comment:
Having two style tags here may cause issues… can we move that `background-image: url({poster})` into the `computed` section?

Reply:
Ups didn't saw that, sorry! Will fix right now.

Reply:
Created a new computed property `videoStyle` and added the `background-image` just to this one (which just concats the `background-image` to the `style` computed property)